bctw-api/src/critterbase/critterbase_api.ts

```diff
@@ -0,0 +1,31 @@
+import axios, { AxiosError, AxiosResponse, Method } from "axios";
+import { formatAxiosError } from "../utils/error";
+
+
+const critterBaseRequest = async (method: Method, url: string, body: any = {}) => {
+    let errStr = '';
+    const header = {headers: { 'API-KEY' : process.env.CRITTERBASE_API_KEY }}
```

Comment:
the header is now included in the exported variable for the axios instance.
critterbase.get etc